pandas/tests/extension/test_arrow.py

```diff
@@ -1026,6 +1026,21 @@ def _get_arith_xfail_marker(self, opname, pa_dtype):
 
         return mark
 
+    def test_round(self, data, request):
+        mark = pytest.mark.xfail(
```

Comment:
nitpick can you define this inside the `if` check.

should we patch the pyarrow behavior?